src/components/NavigationDocs.jsx

```diff
@@ -133,6 +133,13 @@ export const docsNavigation = [
                  { title: 'Manage DNS in your network', href: '/how-to/manage-dns-in-your-network' },
              ]
          },
+         {
+            title: 'Network Flow Logs',
+            isOpen: false,
+            links: [
+                { title: 'Network Flow Logs', href: '/how-to/netflow' },
+            ]
+        },
```

Comment:
There is no need for a separate section. 
The naming is wrong. We have activity section with Audit Events and Traffic Events now.

Reply:
will merge with Bethuel's changes and use the correct section after changes is done.